v20.2/data-types.md

```diff
@@ -47,7 +47,7 @@ CockroachDB supports explicit type conversions using the following methods:
 
 - Other [built-in conversion functions](functions-and-operators.html) when the type is not a SQL type, for example `from_ip()`, `to_ip()` to convert IP addresses between `STRING` and `BYTES` values.
 
- CockroachDB also supports implicit casting from string literals to `INT` and `DECIMAL` [`ARRAY`](array.html)s, where appropriate. For an example, see [Implicit casting to `INT` and `DECIMAL` `ARRAY`s](array.html#implicit-casting-to-int-and-decimal-arrays).
+ CockroachDB also supports implicit casting from string literals to all data types except [`BYTES`](bytes.html), [`ENUM`](enum.html), [`JSONB`](jsonb.html), and [`SERIAL`](serial.html). For an example, see [Implicit casting to `ARRAY`s](array.html#implicit-casting-to-arrays).
```

Comment:
should be "to arrays of all data types"

also, Box2D, Geometry, and Geography types are not supported